### DISCUSSION TITLE
Error in run_plot_calibration

### DISCUSSION THREAD

**Discussion by Guo-icloud** [iiasa/CWatM#149] [Q&A]
Dear Professor,

Hi! After running the 2. Run_plot_calibration.bat, I get the following error, and I didn’t find the solution for this. Could you give me some suggestions?
![error in run_plot_calibration pic](https://github.com/user-attachments/assets/870d9265-d353-4c5a-b379-086c8df65fc1)

Best wishes

**Comment by Guo-icloud** [comment]
Hello, 

I've resolved this problem by renaming the columns in the "observed_data" file, and adjusting the date format as follows:
<img width="151" alt="observed_data_format" src="https://github.com/user-attachments/assets/95edcdbe-2488-46cf-affc-64494d3a814e">

Then the plot now runs successfully. :)

Best wishes,
Xinran

**Reply by leemiaotang** [reply]
good
